src/components/common/slider/slider-bar-handle.js

```diff
@@ -76,7 +77,13 @@ export default class SliderHandle extends Component {
 
   mousemove = e => {
     e.preventDefault();
-    this.props.sliderBarListener(e.movementX);
+    if (e.movementX === 0) {
+      const deltaX = e.clientX - this.prevX;
```

Comment:
Any reason we're using clientX instead of screenX as specified in the spec: https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/movementX ?

Reply:
Perhaps consider de-duplicating these two implementations to increase code re-use.
This also could read a bit like a React hook, if you re-organize it.

Reply:
Read about the difference. Looks like they are both fine in our use case. It's unlikely people change their screen resolution, or move the  viewport while dragging.

__screenX and screenY__
Relative to the top left of the physical screen/monitor, this reference point only moves if you increase or decrease the number of monitors or the monitor resolution.

__clientX and clientY__
Relative to the upper left edge of the content area (the viewport) of the browser window. This point does not move even if the user moves a scrollbar from within the browser.